04-onion-routing.md

```diff
@@ -258,6 +258,7 @@ The writer:
 
 The reader:
   - MUST return an error if `amt_to_forward` or `outgoing_cltv_value` are not present.
+  - MUST return an error if it is not the final node and `short_channel_id` is not present.
```

Comment:
should we also return an error if a node is the final hop and `short_channel_id` is present?